clients/python/girder_client/__init__.py

```diff
@@ -88,6 +88,21 @@ def __str__(self):
         return super(HttpError, self).__str__() + '\nResponse text: ' + self.responseText
 
 
+class DownloadProgressReporter(object):
```

Comment:
We should make this an internal symbol, and change its name to something like `_NoopProgressReporter` to indicate that it does nothing.